bin/varnishd/cache/cache_vrt_vcl.c

```diff
@@ -336,6 +336,8 @@ VRT_Assign_Backend(VCL_BACKEND *dst, VCL_BACKEND src)
 	AN(dst);
 	CHECK_OBJ_ORNULL((*dst), DIRECTOR_MAGIC);
 	CHECK_OBJ_ORNULL(src, DIRECTOR_MAGIC);
+	if (*dst == src)
```

Comment:
I think you should just push this one to master

Reply:
b825f55f348595cfc24f5eee92b86ecf2e880924